src/main/kotlin/com/sourcegraph/cody/ui/CodyToolWindowFactory.kt

```diff
@@ -334,7 +335,9 @@ class WebUIProxy(private val host: WebUIHost, private val browser: JBCefBrowserB
       val browser =
           JBCefBrowserBuilder()
               .apply {
-                setOffScreenRendering(true)
+                val isOffScreenRenderingSupported =
+                    ApplicationInfo.getInstance().build.baselineVersion >= 224
```

Comment:
did you verify the versions in between? does it change exactly in 224?

Reply:
Looks like so, 2024.1 works, and e.g. 2023.3 does not.